keyboards/ergodox_ez/ergodox_ez.c

```diff
@@ -269,68 +269,39 @@ const is31_led g_is31_leds[DRIVER_LED_TOTAL] = {
-rgb_led g_rgb_leds[DRIVER_LED_TOTAL] = {
-
-    /*{row | col << 4}
-      |             {x=0..224, y=0..64}
-      |              |                    flags
-      |              |                    | */
-    {{ 8|(0<<4)},   {17.2* 8, 12.8*0}, 4}, // LED 1 on right > Key 6
-    {{ 9|(0<<4)},   {17.2* 9, 12.8*0}, 4}, // LED 2 > Key 7
-    {{10|(0<<4)},   {17.2*10, 12.8*0}, 4}, // LED 3 > Key 8
-    {{11|(0<<4)},   {17.2*11, 12.8*0}, 4}, // LED 4 > Key 9
-    {{12|(0<<4)},   {17.2*12, 12.8*0}, 4}, // LED 5 > Key 0
-
-    {{ 8|(1<<4)},   {17.2* 8, 12.8*1}, 4}, // LED 6
-    {{ 9|(1<<4)},   {17.2* 9, 12.8*1}, 4}, // LED 7
-    {{10|(1<<4)},   {17.2*10, 12.8*1}, 4}, // LED 8
-    {{11|(1<<4)},   {17.2*11, 12.8*1}, 4}, // LED 9
-    {{12|(1<<4)},   {17.2*12, 12.8*1}, 4}, // LED 10
-
-    {{ 8|(2<<4)},   {17.2* 8, 12.8*2}, 4}, // LED 11
-    {{ 9|(2<<4)},   {17.2* 9, 12.8*2}, 4}, // LED 12
-    {{10|(2<<4)},   {17.2*10, 12.8*2}, 4}, // LED 13
-    {{11|(2<<4)},   {17.2*11, 12.8*2}, 4}, // LED 14
-    {{12|(2<<4)},   {17.2*12, 12.8*2}, 4}, // LED 15
-
-    {{ 8|(3<<4)},   {17.2* 8, 12.8*3}, 4}, // LED 16
-    {{ 9|(3<<4)},   {17.2* 9, 12.8*3}, 4}, // LED 17
-    {{10|(3<<4)},   {17.2*10, 12.8*3}, 4}, // LED 18
-    {{11|(3<<4)},   {17.2*11, 12.8*3}, 4}, // LED 19
-    {{12|(3<<4)},   {17.2*12, 12.8*3}, 4}, // LED 20
-
-    {{ 9|(4<<4)},   {17.2* 9, 12.8*4}, 1}, // LED 21
-    {{10|(4<<4)},   {17.2*10, 12.8*4}, 1}, // LED 22
-    {{11|(4<<4)},   {17.2*11, 12.8*4}, 1}, // LED 23
-    {{12|(4<<4)},   {17.2*12, 12.8*4}, 1}, // LED 24
-
-    {{ 5|(0<<4)},   {17.2* 5, 12.8*0}, 4}, // LED 1 on left > Key 5
-    {{ 4|(0<<4)},   {17.2* 4, 12.8*0}, 4}, // LED 2 > Key 4
-    {{ 3|(0<<4)},   {17.2* 3, 12.8*0}, 4}, // LED 3 > Key 3
-    {{ 2|(0<<4)},   {17.2* 2, 12.8*0}, 4}, // LED 4 > Key 2
-    {{ 1|(0<<4)},   {17.2* 1, 12.8*0}, 4}, // LED 5 > Key 1
-
-    {{ 5|(1<<4)},   {17.2* 5, 12.8*1}, 4}, // LED 6
-    {{ 4|(1<<4)},   {17.2* 4, 12.8*1}, 4}, // LED 7
-    {{ 3|(1<<4)},   {17.2* 3, 12.8*1}, 4}, // LED 8
-    {{ 2|(1<<4)},   {17.2* 2, 12.8*1}, 4}, // LED 9
-    {{ 1|(1<<4)},   {17.2* 1, 12.8*1}, 4}, // LED 10
-
-    {{ 5|(2<<4)},   {17.2* 5, 12.8*2}, 4}, // LED 11
-    {{ 4|(2<<4)},   {17.2* 4, 12.8*2}, 4}, // LED 12
-    {{ 3|(2<<4)},   {17.2* 3, 12.8*2}, 4}, // LED 13
-    {{ 2|(2<<4)},   {17.2* 2, 12.8*2}, 4}, // LED 14
-    {{ 1|(2<<4)},   {17.2* 1, 12.8*2}, 4}, // LED 15
-
-    {{ 5|(3<<4)},   {17.2* 5, 12.8*3}, 4}, // LED 16
-    {{ 4|(3<<4)},   {17.2* 4, 12.8*3}, 4}, // LED 17
-    {{ 3|(3<<4)},   {17.2* 3, 12.8*3}, 4}, // LED 18
-    {{ 2|(3<<4)},   {17.2* 2, 12.8*3}, 4}, // LED 19
-    {{ 1|(3<<4)},   {17.2* 1, 12.8*3}, 4}, // LED 20
-
-    {{ 4|(4<<4)},   {17.2* 4, 12.8*4}, 1}, // LED 21
-    {{ 3|(4<<4)},   {17.2* 3, 12.8*4}, 1}, // LED 22
-    {{ 2|(4<<4)},   {17.2* 2, 12.8*4}, 1}, // LED 23
-    {{ 1|(4<<4)},   {17.2* 1, 12.8*4}, 1}, // LED 24 > Key Hack
-};
+led_config_t g_led_config = { {
+    { NO_LED, NO_LED, NO_LED, NO_LED, NO_LED, NO_LED },
```

Comment:
Ooooh, wait, this is based off of the physical switch matrix, isn't it? :)

Reply:
correct

Reply:
And just to clarify, that makes me very excited and happy. :)

Because it makes it easier to understand (to me), and opens the ability to match/identify key and led relationships